### PR TITLE
Feature/cplp 467 bp data structure changes

### PR DESCRIPTION
**Pull request by fatihayyildiz**
Partner network redux store implementation changed
Pagination implemented for table result
Some small style alignments implemented

### CODE REVIEW

File: cx-portal/src/components/pages/PartnerNetwork/index.tsx [hunk 1]
```diff
@@ -139,6 +157,20 @@ const PartnerNetwork = () => {
...
         />
       </div>
+      <div className="load-more-button-container">
+        {businessPartners.totalElements > pageSize * currentPage && (
+          <Button
+            size="medium"
+            onClick={() =>
+              setCurrentPage((prevState) => {
+                return prevState + 1
+              })
```
Comment (oyo) [inline cx-portal/src/components/pages/PartnerNetwork/index.tsx:167]: this should be enough
`onClick={() => setCurrentPage((prevState) => prevState + 1)}`

Comment (fatihayyildiz) [reply cx-portal/src/components/pages/PartnerNetwork/index.tsx:167]: Sure, I will update PR according to these hints

Comment (fatihayyildiz) [reply cx-portal/src/components/pages/PartnerNetwork/index.tsx]: I updated PR, could you please check ?

Comment (oyo) [reply cx-portal/src/components/pages/PartnerNetwork/index.tsx]: looks good now - thanks for the update... =D

**Pull request merged**